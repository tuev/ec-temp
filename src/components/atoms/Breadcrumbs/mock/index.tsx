import React, { FC } from 'react'
import Typography from '@material-ui/core/Typography'
import Link from '@material-ui/core/Link'

const items: FC = () => (
  <>
    <Link color="inherit" href="/">
      Home
    </Link>
    <Link color="inherit" href="/getting-started/installation/">
      All Categories
    </Link>
    <Typography color="textPrimary">
      Men&apos;s Clothing & Accessories
    </Typography>
  </>
)

export default items
