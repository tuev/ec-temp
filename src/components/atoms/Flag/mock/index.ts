import enFlag from './icon-flag-gbr.png'

export { enFlag }
