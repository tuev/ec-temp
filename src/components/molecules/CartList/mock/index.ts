import mockImage from './img.png'
import { CartItemProps } from '../CartList.types'

const cartItems: CartItemProps[] = [
  {
    image: mockImage,
    size: 'M',
    quantity: 1,
    name: 'Product name 1',
    id: '123123',
    price: 32,
  },
  {
    image: mockImage,
    size: 'L',
    quantity: 2,
    name: 'Product name 2',
    id: '123122',
    price: 32,
  },
  {
    image: mockImage,
    size: 'XL',
    quantity: 3,
    name: 'Product name 3',
    id: '123124',
    price: 32,
  },
]

export default cartItems
