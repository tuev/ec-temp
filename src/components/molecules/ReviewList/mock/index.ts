export const reviewList = [
  {
    id: 1,
    rating: 3,
    fullname: 'Pablo Permins',
    dateCreated: 'an hour ago',
    reviewText:
      'Got this through the post the other day and right from opening the packet I knew this was quality, put it on and I was right!! Well done',
    likeCount: 0,
    commentCount: 20,
  },
]
