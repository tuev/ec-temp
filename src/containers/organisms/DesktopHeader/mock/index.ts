const HeaderItem = [
  {
    key: 'women',
    href: '/productlist?category=women',
    title: 'Women',
  },
  {
    key: 'men',
    href: '/productlist?category=men',
    title: 'Men',
  },
  {
    key: 'shoe',
    href: '/productlist?category=shoe',
    title: 'Shoe',
  },
  {
    key: 'brands',
    href: '/productlist',
    title: 'Brands',
  },
]

export default HeaderItem
