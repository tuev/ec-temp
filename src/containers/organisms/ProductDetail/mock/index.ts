import { ProductDetailProps } from './../ProductDetail.types'
import img1 from './img2.jpg'
import img2 from './img3.jpg'
import img3 from './img4.jpg'

export const itemInfo: ProductDetailProps = {
  id: '123468761283',
  images: [img1, img2, img3],
  breadcrumbLinks: [{ href: '/', title: 'Home' }],
  activeBreadcrumb: 'product',
  rating: 1,
  name: 'Product name',
  price: '2$',
  colors: ['black', 'blue', 'red', 'yellow', 'green'],
  sizes: ['L', 'M', 'S', 'XL'],
  quantity: 1,
}
