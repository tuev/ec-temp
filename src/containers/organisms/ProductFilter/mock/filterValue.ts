import { IProductFilterOptions } from '../ProductFilter.types'
const filterOptions: IProductFilterOptions = {
  brand: {
    brand1: {
      label: 'Brand 1',
      value: true,
    },
    brand2: {
      label: 'Brand 2',
      value: false,
    },
  },
  color: ['black', 'green', 'yellow', 'red', 'blue', 'orange'],
  price: [5, 95],
  size: ['S', 'M', 'L', 'XS', 'XXL'],
}

export { filterOptions }
