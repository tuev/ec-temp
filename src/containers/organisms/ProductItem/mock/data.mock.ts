import { ProductItemProps } from '../ProductItem.types'

const data: ProductItemProps[] = [
  {
    price: '1$',
    name: 'name 1',
    id: '1',
    colors: ['green', 'red', 'yellow', 'blue'],
    sizes: ['L', 'M', 'XS', 'S'],
  },
  {
    price: '1$',
    name: 'name 1',
    id: '1',
    colors: ['green', 'red', 'yellow', 'blue'],
    sizes: ['L', 'M', 'XS', 'S'],
  },
  {
    price: '1$',
    name: 'name 1',
    id: '1',
    colors: ['green', 'red', 'yellow', 'blue'],
    sizes: ['L', 'M', 'XS', 'S'],
  },
  {
    price: '1$',
    name: 'name 1',
    id: '1',
    colors: ['green', 'red', 'yellow', 'blue'],
    sizes: ['L', 'M', 'XS', 'S'],
  },
  {
    price: '1$',
    name: 'name 1',
    id: '1',
    colors: ['green', 'red', 'yellow', 'blue'],
    sizes: ['L', 'M', 'XS', 'S'],
  },
  {
    price: '1$',
    name: 'name 1',
    id: '1',
    colors: ['green', 'red', 'yellow', 'blue'],
    sizes: ['L', 'M', 'XS', 'S'],
  },
  {
    price: '1$',
    name: 'name 1',
    id: '1',
    colors: ['green', 'red', 'yellow', 'blue'],
    sizes: ['L', 'M', 'XS', 'S'],
  },
]

export default data
