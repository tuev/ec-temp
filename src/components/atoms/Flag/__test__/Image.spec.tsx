import React from 'react'
import { shallow } from 'enzyme'
import { Flag } from '..'
import toJson from 'enzyme-to-json'
import { enFlag } from '../mock'

describe('Flag', () => {
  it('Flag no width and height attribute', () => {
    const AppFlag = shallow(<Flag src={enFlag} alt="flag" />)

    expect(toJson(AppFlag)).toMatchSnapshot()
  })
})
