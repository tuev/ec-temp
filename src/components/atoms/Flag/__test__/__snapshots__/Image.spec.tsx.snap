// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flag Flag no width and height attribute 1`] = `
<styled.img
  alt="flag"
  src="icon-flag-gbr.png"
/>
`;
